### PR TITLE
Make FileListener respect .nomedia

### PR DESCRIPTION
**Pull request by marcelklehr**

### CODE REVIEW

File: test/ClassifierTest.php [hunk 1]
```diff
@@ -108,6 +108,35 @@ public function testSchedulerJob() : void {
...
+	 * @return void
+	 * @throws \OCP\DB\Exception
+	 * @throws \OCP\Files\InvalidPathException
+	 * @throws \OCP\Files\NotFoundException
+	 * @throws \OCP\Files\NotPermittedException
+	 */
+	public function testFileListener(string $ignoreFileName) : void {
+		$this->config->setAppValue('recognize', 'imagenet.enabled', 'true');
+
+		$this->testFile = $this->userFolder->newFile('/alpine.jpg', file_get_contents(__DIR__.'/res/alpine.JPG'));
```
Comment (marcelklehr) [inline test/ClassifierTest.php:122]: @juliushaertl Shouldn't this trigger the file event listener?

Comment (juliushaertl) [reply test/ClassifierTest.php:122]: Yes, though one thing I'm unsure is if the test boostrap properly triggers the app loading in terms of that the Application register/boot methods are called. Maybe try to set an xdebug breakpoint or dd in the Application class when running the tests to see if that one actually gets registered.

Comment (marcelklehr) [reply test/ClassifierTest.php:122]: The problem was storage ID vs numeric storage ID :see_no_evil:

**Comment by marcelklehr** [comment]
nc master failure is unrelated

**Pull request closed without merging**